components/@ui/Select.jsx

```diff
@@ -126,7 +126,9 @@ const Select = ({ value, onChange, options, multiple, defaultLabel }) => {
         {options.map((option) => (
           <li
             className={`text-sm px-3 py-2 hover:bg-gray-500 hover:text-white ${
-              darkMode ? 'bg-gray-900 text-gray-400 border-gray-500 ' : ''
+              theme === 'dark'
```

Comment:
Using ternary operators (here and elsewhere below) is great for booleans, but not ideal if there are more than two states to toggle across (difficult to read, easy to introduce errors). So you might implement this more verbosely to avoid needing to change the general flow of the logic if/when/once additional themes are introduced.

Reply:
@uniqueg  Noted. Thanks for the review.

Reply:
@uniqueg for inline styles, we should use ternary operators.  
cc: @git-anurag-hub